app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -40,19 +39,18 @@ def sync_repos_to_capsule(capsule_content, repository_ids)
           concurrence do
             repository_ids.each do |repo_id|
               sequence do
-                repo = ::Katello::Repository.where(:pulp_id => repo_id).first ||
-                       ::Katello::ContentViewPuppetEnvironment.where(:pulp_id => repo_id).first
-                if repo && repo.content_type != "yum"
-                  # we unassociate units in non-yum repos in order to avoid version conflicts
-                  # during publish. (i.e. two versions of a puppet module in the same repo)
+                repo = ::Katello::Repository.find_by(pulp_id: repo_id)
+                if repo && ["docker", "file", "ostree"].include?(repo.content_type)
```

Comment:
@jlsherrill Do you think this would make more sense as `['yum', 'puppet'].exclude?(repo.content_type)`?